### PR TITLE
refactor(solo): Convert RESM to NESM

### PR DESCRIPTION
**Pull request by kriskowal**
Refs #527

### CODE REVIEW

File: packages/solo/src/web.js [hunk 1]
```diff
@@ -1,19 +1,15 @@
-/* global __dirname require setTimeout clearTimeout setInterval clearInterval */
+/* global setTimeout clearTimeout setInterval clearInterval */
 // Start a network service
 import path from 'path';
 import http from 'http';
 import { createConnection } from 'net';
 import express from 'express';
 import WebSocket from 'ws';
 import anylogger from 'anylogger';
+import morgan from 'morgan';
```
Comment (michaelfig) [inline packages/solo/src/web.js:9]: I had to test this manually, but yes, it works.  Thanks for the simplification!

**Review by michaelfig — CHANGES_REQUESTED** [review]
https://github.com/Agoric/agoric-sdk/pull/3619/checks?check_run_id=3266996850#step:8:205 indicates a problem with:

```console
$ rg entrypoint.cjs
[...]
./packages/agoric-cli/lib/start.js
116:    agSolo = path.resolve(__dirname, '../../solo/src/entrypoint.cjs');
```

**Review by michaelfig — APPROVED** [review]
CI will keep you honest.  No need to request changes.

**Review by michaelfig — CHANGES_REQUESTED** [review]
No, CI doesn't keep you honest when targeting a PR on a branch.

**Comment by kriskowal** [comment]
> No, CI doesn't keep you honest when targeting a PR on a branch.

CI will keep me honest before the code reaches master. The stacked diffs technique I’m using allows me to land the entire stack quickly using merge commits, once the top of the stack is immediately on top of the latest master. Once I have the whole stack reviewed, I can get the entire stack through CI in one round trip (while under a single Keybase lock on master), then land each PR without triggering a re-run of CI between each.

With an approval, I could accidentally merge this into the base branch, but that will have to pass CI on top of the most recent master too.

**Comment by kriskowal** [comment]
Now at the head of the queue 😇

**Review by michaelfig — APPROVED** [review]

**Pull request merged**